services/webnn/ort/scoped_ort_types.h

```diff
@@ -97,7 +111,9 @@ class ScopedOrtNode {
 
 class ScopedOrtModel {
  public:
-  ScopedOrtModel();
+  explicit ScopedOrtModel();
```

Comment:
It seems these  `explicit` are unnecessary, `ScopedOrt**` constructor has no parameters.

Reply:
👍 Removed